.github/workflows/performance.yml

```diff
@@ -32,17 +32,19 @@ permissions: {}
 jobs:
   # Runs the performance test suite.
   performance:
-    name: Performance tests ${{ matrix.memcached && '(with memcached)' || '' }}
-    uses: WordPress/wordpress-develop/.github/workflows/reusable-performance.yml@trunk
+    name: Performance tests / ${{ matrix.multisite && 'Multisite' || 'Single' }}${{ matrix.memcached && ' (memcached)' || '' }}
```

Comment:
To help cut down on the long job names in other contexts than the workflow view, only include the first part of the name in the calling workflow. Everything after the `/` should only be in the called workflow.

Reply:
But then I get "true, false" everywhere:

<img width="312" alt="Screenshot 2024-12-19 at 14 07 09" src="https://github.com/user-attachments/assets/db16b820-294a-403e-83c0-a5a897dd8362" />

Reply:
Getting closer:

<img width="682" alt="Screenshot 2024-12-19 at 14 28 53" src="https://github.com/user-attachments/assets/e60920e2-6ffb-4979-96cc-2f2202b7c698" />
<img width="354" alt="Screenshot 2024-12-19 at 14 28 35" src="https://github.com/user-attachments/assets/c83dd6ef-4bcc-46af-8732-e16e8d01a587" />

Reply:
Hmm, ya that's not great either. https://github.com/WordPress/wordpress-develop/pull/6515/commits/eb88943ed559d84f4a9061b029d8ac5fde10c780 looks much better, though. Do you like that approach? Waiting to see what the summary titles are.

Reply:
The titles in the status checks section still isn't great, but the repetition is needed if we want that kind of nesting.

The "PHP x.x" part is also repeated for the PHP unit tests.

So I think this is as good as it gets.

Reply:
Yes unfortunately one element will always repeat outside of the workflow run view.